include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -1438,9 +1533,11 @@ class FEEvaluationAccess<1,1,Number> : public FEEvaluationBase<1,1,Number>
  *
  *         // Set n_items unit vectors
  *         for (unsigned int j=0; j<dofs_per_cell; ++j)
- *           fe_eval.begin_dof_values()[j]  = VectorizedArray<double>();
+ *           fe_eval.set_dof_value(VectorizedArray<double>(), j);
+ *         VectorizedArray<double> one_value = VectorizedArray<double>();
```

Comment:
I realize that I introduced a bug into this code when switching from the pointer access through `begin_dof_values()` to the access function `set_dof_value(i)`. One wants to have one entry in the vectorized array to one and the others to zero. Please check the new code that I will commit soon.